content/facilities/va-facilities/release-notes/2024-12-16.md

```diff
@@ -0,0 +1,4 @@
+We have added a health service:
+- `chaplain`: Chaplain services address spiritual, emotional, moral, religious, and ethical concerns of VA patients.
+
+[View the code change(s)](https://github.com/department-of-veterans-affairs/lighthouse-facilities/pull/599)
```

Comment:
```suggestion
For more information, [review the API's documentation.](https://developer.va.gov/explore/api/va-facilities/docs?version=current)
```
We're avoiding linking out to VA's codebase in release notes, especially since they are visible to the public. Instead, we are directing consumers to the API's documentation.